Dockerfile

```diff
@@ -1,16 +1,29 @@
-FROM alpine:3.9.2
+FROM quay.io/openshift/origin-cli:v4.0.0
 
-ARG kubectlversion=1.12.4
-ARG yqversion=2.2.1
+ARG ocpythonlibver=0.8.6
 
-RUN apk add --no-cache curl bash
 RUN \
-  cd /usr/local/bin             && \
-  echo "Grabbing binaries..." && \
-  curl -sLO https://storage.googleapis.com/kubernetes-release/release/v${kubectlversion}/bin/linux/amd64/kubectl && \
-  curl -sLO https://github.com/mikefarah/yq/releases/download/${yqversion}/yq_linux_amd64 && \
-  mv /usr/local/bin/yq_linux_amd64 /usr/local/bin/yq && \
-  chmod +x /usr/local/bin/kubectl /usr/local/bin/yq
-ADD fetch.sh /usr/local/bin
+  cd /tmp && \
+  curl -L https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
+  python /tmp/get-pip.py && \
+  pip install -U setuptools
```

Comment:
The following should remove the need to uninstall and reinstall chardet and requests

`pip install -U setuptools urllib3==1.24.1 chardet==3.0.4 requests==2.21.0`

Reply:
`pip install setuptools==40.8.0 urllib3==1.24.1 chardet==3.0.4 requests==2.21.0`

Would set the version of setuptools and remove the `-U` flag

often people add a requirements.txt file with the python deps then call `pip install -r requirements.txt` but i think this is fine since there isn't many deps.